README.rst

```diff
@@ -11,49 +11,169 @@
 
 |
 
+
 **Ray is a fast and simple framework for building and running distributed applications.**
 
+Ray is packaged with the following libraries for accelerating machine learning workloads:
+
+- `Tune`_: Scalable Hyperparameter Tuning
+- `RLlib`_: Scalable Reinforcement Learning
+- `Distributed Training <https://ray.readthedocs.io/en/latest/distributed_training.html>`__
 
-Ray is easy to install: ``pip install ray``
+Install Ray with: ``pip install ray``. For nightly wheels, see the `Installation page <https://ray.readthedocs.io/en/latest/installation.html>`__.
 
-Example Use
+Quick Start
 -----------
 
-+------------------------------------------------+----------------------------------------------------+
-| **Basic Python**                               | **Distributed with Ray**                           |
-+------------------------------------------------+----------------------------------------------------+
-|.. code-block:: python                          |.. code-block:: python                              |
-|                                                |                                                    |
-|  # Execute f serially.                         |  # Execute f in parallel.                          |
-|                                                |                                                    |
-|                                                |  @ray.remote                                       |
-|  def f():                                      |  def f():                                          |
-|      time.sleep(1)                             |      time.sleep(1)                                 |
-|      return 1                                  |      return 1                                      |
-|                                                |                                                    |
-|                                                |                                                    |
-|                                                |  ray.init()                                        |
-|  results = [f() for i in range(4)]             |  results = ray.get([f.remote() for i in range(4)]) |
-+------------------------------------------------+----------------------------------------------------+
-
-
-Ray comes with libraries that accelerate deep learning and reinforcement learning development:
-
-- `Tune`_: Hyperparameter Optimization Framework
-- `RLlib`_: Scalable Reinforcement Learning
-- `Distributed Training <http://ray.readthedocs.io/en/latest/distributed_training.html>`__
+.. code-block:: python
```

Comment:
Maybe add some additional line above this one like "Execute Python functions in parallel."